core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2155,14 +2173,17 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
 //   @Message(value = "Another partition or topology changed for while completing partial completed transaction. Retrying later.", id = 630)
 //   void topologyChangedPartitionHandlingTxCompletion();
 
-   @Message(value = "Cross-Site state transfer mode cannot be null.", id = 633)
+   @Message(value = "Cross-site state transfer mode cannot be null.", id = 633)
+   @Description("The mode attribute for state transfer configuration must have a value of AUTO or MANUAL. Modify the cache configuration with a valid value for the mode attribute.")
    CacheConfigurationException invalidXSiteStateTransferMode();
 
-   @Message(value = "Cross-Site automatic state transfer is not compatible with SYNC backup strategy.", id = 634)
+   @Message(value = "Cross-site automatic state transfer is not compatible with SYNC backup strategy.", id = 634)
+   @Description("Automatic state transfer is not possible if the backup strategy for cross-site replication is synchronous. Modify the cache configuration and set the state transfer mode to MANUAL. Alternatively you can change the backup strategy to use asynchronous mode.")
    CacheConfigurationException autoXSiteStateTransferModeNotAvailableInSync();
 
    @LogMessage(level = WARN)
    @Message(value = "[%s] Failed to receive a response from any nodes. Automatic cross-site state transfer to site '%s' is not started.", id = 635)
+   @Description("State transfer operations start automatically only if nodes at the backup location are available. Check that all clusters that participate in cross-site replication are online.")
```

Comment:
This is an exception from the local cluster. The coordinator checks all local nodes to see if the state transfer is required but it got an exception instead.